### PR TITLE
refactor: move snippets from ertp/guide

### LINKED ISSUE #336: Test Code Snippets in ERTP/guide files
See parent issue #335

### PR DESCRIPTION
**Pull request by katelynsills**
This PR tests the code snippets in the ERTP guide. A number of small fixes were found - nothing alarming, but I think it's a good thing we are testing.

- [x] amount-math.md
- [x] amounts.md
- [x] issuers-and-mints.md
- [x] purses-and-payments.md
- [x] README.md

Closes #336

### CODE REVIEW

File: main/ertp/guide/amounts.md [hunk 1]
```diff
@@ -13,9 +13,7 @@ to get information about and otherwise manipulate an `amount` as a whole.
 
 However, you can also make an `amount` as an object literal by making a record of
 a `brand` and a `value`. While `amountMath.make()` is recommended for proper object-oriented programming (it also checks the `brand`'s validity), this produces the same result:
-```js
-const newAmount = { brand: quatloosBrand, value: 5 }
-```
+<<< @/snippets/ertp/guide/test-amounts.js#manualMake
```
Comment (Chris-Hibbert) [inline main/ertp/guide/amounts.md:16]: This snippet is broken.  It displays literally.

```
this produces the same result: <<< @/snippets/ertp/guide/test-amounts.js#manualMake amount has two properties:
```

File: snippets/ertp/guide/test-issuers-and-mints.js [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+  t.is(quatloosBrand, myQuatloosBrand);
+});
+
+test('ertp guide issuers and mints getAllegedName', async t => {
+  // #region getAllegedName
+  const { issuer: quatloosIssuer } = makeIssuerKit('quatloos');
+  const quatloosIssuerAllegedName = quatloosIssuer.getAllegedName();
+  // quatloosIssuerAllegedName === 'quatloos'
+  t.is(quatloosIssuerAllegedName, 'quatloos');
+  // #endregion getAllegedName
```
Comment (Chris-Hibbert) [inline snippets/ertp/guide/test-issuers-and-mints.js:55]: Not clear the `t.is()` should be within the snippet. Many other places, there's a comment asserting something about the results without a test to validate it being in the snippet.

**Review by Chris-Hibbert — CHANGES_REQUESTED** [review]
Thanks for doing this. It's nice to have all this sample code validated.

One error, one question about consistency.

File: snippets/ertp/guide/test-issuers-and-mints.js (hunk 2 above)
Comment (katelynsills) [reply snippets/ertp/guide/test-issuers-and-mints.js:55]: I'll remove. I was considering including more test assertions in the code examples but there's no need to have a comment and an assertion

**Review by Chris-Hibbert — APPROVED** [review]

**Pull request merged**